misc/plugutils.c

```diff
@@ -484,19 +484,21 @@ check_kb_inconsistency_log (kb_t lkb, const char *name)
     {
     case -3:
       current_scan_id = kb_item_get_str (kb, ("internal/scanid"));
-      g_warning (
-        "%s: scan_id (%s) does not match global scan_id (%s); will not "
-        "store %s in kb.",
-        __func__, current_scan_id, get_scan_id (), name);
+      g_error ("%s: scan_id (%s) does not match global scan_id (%s); abort to "
```

Comment:
g_error should not be use for runtime errors (see https://docs.gtk.org/glib/error-reporting.html). 
I found also, that in earlier glib version it raised SIGABORT. From version 2.39 it raises SIGTRAP.
Probably we should just use g_warning here.